api/spec/v1.yaml

```diff
@@ -1959,6 +1949,30 @@ components:
       description: |
         A runtime transaction.
 
+    RuntimeAccount:
+      type: object
+      required: [address, address_preimage, runtime, sdk_balance, stats]
+      properties:
+        address:
+          type: string
+          description: The staking address for this account.
+          example: *staking_address_1
+        address_preimage:
+          $ref: '#/components/schemas/AddressPreimage'
+        sdk_balance:
```

Comment:
I know I named this, but now I'm wondering if it's too technical. Runtime users don't know about the SDK. Is `runtime_balance` better, or too vague? Does anybody have other ideas?

Reply:
I like `runtime_balance` better; changed to that for now. I'd be ok using `balance` as well